### PR TITLE
docs(compute-samples): added parallel testing support

### PR DESCRIPTION
**Pull request by Sita04**
Added parallel testing support for samples with mvn surefire and JUnit jupiter.

### CODE REVIEW

File: compute/cloud-client/src/main/java/compute/CreateEncryptedInstance.java [hunk 1]
```diff
@@ -111,7 +113,8 @@ public static void createEncryptedInstance(String project, String zone, String i
           instancesClient.insertAsync(insertInstanceRequest);
 
       // Wait for the operation to complete.
-      Operation response = operation.get();
+      Operation response = operation.get(3, TimeUnit.MINUTES);
+      ;
```
Comment (Shabirmean) [inline compute/cloud-client/src/main/java/compute/CreateEncryptedInstance.java:117]: Remove extra semi colon

File: compute/cloud-client/src/main/java/compute/CreateInstance.java [hunk 2]
```diff
@@ -105,7 +107,8 @@ public static void createInstance(String project, String zone, String instanceNa
           insertInstanceRequest);
 
       // Wait for the operation to complete.
-      Operation response = operation.get();
+      Operation response = operation.get(3, TimeUnit.MINUTES);
+      ;
```
Comment (Shabirmean) [inline compute/cloud-client/src/main/java/compute/CreateInstance.java:111]: Remove semi colon in all the files where this appears in extra

File: compute/cloud-client/src/test/java/compute/FirewallIT.java [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+  public static boolean isFirewallRuleDeletedByGceEnforcer(String projectId,
+      String firewallRule) throws IOException, ExecutionException, InterruptedException {
+    /* (**INTERNAL method**)
+      This method will prevent test failure if the firewall rule was auto-deleted by GCE Enforcer.
+      (Feel free to remove this method if not running on a Google-owned project.)
+     */
+    try {
+      GetFirewallRule.getFirewallRule(projectId, firewallRule);
+    } catch (NotFoundException e) {
+      System.out.println("Rule already deleted ! ");
```
Comment (Shabirmean) [inline compute/cloud-client/src/test/java/compute/FirewallIT.java:105]: (nit) `deleted ! ` -> `deleted!`

**Review by Shabirmean — COMMENTED** [review]
In general LGTM! Left some nit comments.

Tagging @lesv since I am still a a shadow reviewer

File: compute/cloud-client/src/test/java/compute/InstancesAdvancedIT.java [hunk 4]
```diff
@@ -0,0 +1,284 @@
...
+    MACHINE_NAME_SNAPSHOT_ADDITIONAL = "test-instance-snapa-" + UUID.randomUUID();
+    MACHINE_NAME_SUBNETWORK = "test-instance-subnet-" + UUID.randomUUID();
+    NETWORK_NAME = "global/networks/default";
+    SUBNETWORK_NAME = "regions/us-central1/subnetworks/default";
+
+    TEST_DISK = createSourceDisk();
+    TEST_SNAPSHOT = createSnapshot(TEST_DISK);
+    TEST_IMAGE = createImage(TEST_DISK);
+
+    // Util.cleanUpExistingInstances("test-instance", PROJECT_ID, ZONE);
```
[the remark this answers is not in the document]
Comment (Sita04) [reply compute/cloud-client/src/test/java/compute/InstancesAdvancedIT.java]: Thank you. That was not supposed to be commented out!

**Review by kurtisvg — APPROVED** [review]

**Pull request merged**